src/tools/rbd_mirror/Replayer.cc

```diff
@@ -273,10 +273,14 @@ void Replayer::set_sources(const map<int64_t, set<string> > &images)
 	   images_it != pool_images.end();) {
 	if (stop_image_replayer(images_it->second)) {
 	  pool_images.erase(images_it++);
```

Comment:
While fixing this, can we also switch this (and below) to something like `images_it = pool_images.erase(images_it)`?